worker/src/RTC/RTCP/CompoundPacket.cpp

```diff
@@ -12,17 +12,17 @@ namespace RTC
 
 		CompoundPacket::UniquePtr CompoundPacket::Create()
 		{
-			auto* packet = CompoundPacket::Allocator::Pool.Allocate();
-			CompoundPacket::Allocator::Pool.construct(packet);
+			auto* packet = CompoundPacket::Allocator::Pool.allocate(1);
+			CompoundPacket::AllocatorTraits::construct(CompoundPacket::Allocator::Pool, packet);
 			return UniquePtr(packet);
 		}
 
 		void CompoundPacket::ReturnIntoPool(CompoundPacket* packet)
```

Comment:
Would be better to get rid of this and use smart pointer's capabilities of customizing both deleter and allocator.

Reply:
There is no allocator for `unique_ptr`, but we could use the `deleter` to destroy the instance and deallocate the memory. Can you do that @Hartigan ?

Reply:
@jmillan  currentry it's implemented via partial specialization of ```std::default_delete```

Reply:
I think specialization like that is more confusing/magical, let's use deleter instead.

Reply:
updated